packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -66,22 +67,23 @@ export const files = async ({
     (isWordPluralizable(cellName) ? isPlural(cellName) : options.list) ||
     options.list
 
+  // needed for the singular cell GQL query find by id case
+  try {
+    model = await getSchema(pascalcase(singularize(cellName)))
+    idType = getIdType(model)
+    mockIdValue = idType === 'String' ? "'42'" : 42
```

Comment:
```suggestion
```

Reply:
I also realized that the String flavor of these cells needs tests.